boshio/boshio.go

```diff
@@ -29,19 +31,19 @@ type ranger interface {
 }
 
 //go:generate counterfeiter -o ../fakes/http_client.go --fake-name HTTPClient . httpClient
-type httpClient interface {
+type HTTPClient interface {
```

Comment:
We can take this back to private. No need to have it as public because we are only consuming it here (see other comment about not return an interface).